### ISSUE TITLE
RCTJSONStringify() encountered the following error: Invalid number value (NaN) in JSON write

### ISSUE THREAD

**Issue by var77** [TheWidlarzGroup/react-native-video#792]
Getting the error on iOS 11, temporary solution is to remove 
`  @"seekableDuration": [NSNumber numberWithFloat:CMTimeGetSeconds([self playerItemSeekableTimeRange].duration)],
` in ios/RCTVideo.m:198

**Comment by zwm0800** [comment]
thanks!

**Comment by tiendn** [comment]
Hope to fix soon 👍

**Comment by axelphunter** [comment]
+1

**Comment by Gordeev1** [comment]
Fixed by [commit f73b7](https://github.com/react-native-community/react-native-video/commit/f73b7a04843fb5b17fb4fbedbe593c225594ab3b).
Npm registry does not contain this commit, to apply this changes just install from repo
````
npm i -S https://github.com/react-native-community/react-native-video
````

**Comment by tiendn** [comment]
@Gordeev1 Thanks! It works for me.

**Comment by csotiriou** [comment]
I hope the NPM repo gets updated with this.

**Comment by cobarx** [comment]
This fix is included in at least 2.1.1, not sure about earlier versions.
